test/unit/widgets/forms.test.ts

```diff
@@ -1,1435 +1,1455 @@
+    const store = namedNode('http://example.com/#store')
+    const callbackFunction = () => {}
+    expect(
+      field[ns.ui('Form').uri](
+        document,
+        container,
+        already,
+        subject,
+        form,
+        store,
+        callbackFunction
+      )
+    ).toMatchInlineSnapshot(`
+      <div>
+        <h3
+          style="[object Object]"
+        >
+          [object Object]
+        </h3>
+      </div>
+    `)
+  })
+  // @@ TODO need to double check proper subjects and what gets returned
+  it.skip('returns.. if the subject has already been processed...', () => {
+    const dubSubject = namedNode('subject')
+    const container = document.createElement('container')
+    const already = { dubSubject }
+    const subject = dubSubject
+    const form = namedNode('http://example.com/#form')
+    const store = namedNode('http://example.com/#store')
+    const callbackFunction = () => {}
+    expect(
+      field[ns.ui('Form').uri](
+        document,
+        container,
+        already,
+        subject,
+        form,
+        store,
+        callbackFunction
+      )
+    ).toMatchInlineSnapshot(`
+      <div>
+        <h3
+          style="[object Object]"
+        >
+          [object Object]
+        </h3>
+      </div>
+    `)
+  })
+})
+
+describe('Options field', () => {
+  it('exists', () => {
+    expect(field[ns.ui('Options').uri]).toBeInstanceOf(Object)
+  })
+  it('runs', () => {
+    const container = document.createElement('div')
+    const already = {}
+    const subject = namedNode('http://example.com/#this')
+    const form = namedNode('http://example.com/#form')
+    const store = namedNode('http://example.com/#store')
+    const callbackFunction = () => {}
+    expect(
+      field[ns.ui('Options').uri](
+        document,
+        container,
+        already,
+        subject,
+        form,
+        store,
+        callbackFunction
+      )
+    ).toMatchInlineSnapshot('<div />')
+  })
+})
+
+describe('Multiple field', () => {
+  it('exists', () => {
+    expect(field[ns.ui('Multiple').uri]).toBeInstanceOf(Object)
+  })
+  it.skip('runs', () => {
+    const container = document.createElement('div')
+    const already = {}
+    const subject = namedNode('http://example.com/#this')
+    const form = namedNode('http://example.com/#form')
+    const store = namedNode('http://example.com/#store')
+    const callbackFunction = () => {}
+    expect(
+      field[ns.ui('Multiple').uri](
+        document,
+        container,
+        already,
+        subject,
+        form,
+        store,
+        callbackFunction
+      )
+    ).toMatchInlineSnapshot(`
+      <div>
+        <h3
+          style="[object Object]"
+        >
+          [object Object]
+        </h3>
+      </div>
+    `)
+  })
+})
+
+describe('fieldParams', () => {
+  it('exists', () => {
+    expect(fieldParams).toBeInstanceOf(Object)
+  })
+})
+
+describe('ColorField params', () => {
```

Comment:
Maybe you should group this under describe('fieldParams') block? Since they are all related to `fieldParams`, makes it easier to scan the blocks of tests.

Reply:
yeah, they're moved to a separate file in the fieldParams PR